io.github.torbuntu.Leikr.json

```diff
@@ -0,0 +1,42 @@
+{
+	"id": "io.github.torbuntu.Leikr",
+	"branch": "1.0",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"command": "/app/Leikr/Leikr",
+	"sdk": "org.freedesktop.Sdk",
+	"sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk"],
+	"modules": [{
+			"name": "openjdk",
+			"buildsystem": "simple",
+			"build-commands": ["/usr/lib/sdk/openjdk/install.sh"]
+		}, {
+			"name": "leikr",
+			"buildsystem": "simple",
+			"build-commands": [
+				"install --directory --mode=0755 /app/Leikr/",
+				"tar --directory=/app/Leikr/ --extract --file=Leikr.tar.xz",
+				"install -D --mode=0644 --target-directory=/app/share/applications/ /app/Leikr/Data/io.github.torbuntu.Leikr.desktop",
+				"install -D --mode=0644 /app/Leikr/Data/Logo/io.github.torbuntu.Leikr.png /app/share/icons/hicolor/128x128/apps/io.github.torbuntu.Leikr.png" 
+			],
+			"sources": [
+				{
+					"type": "file",
+					"url": "https://github.com/Torbuntu/Leikr/releases/download/1.0.1/Leikr.tar.xz",
+					"sha256": "42096f4df487bff8bfe45e5b5fa02b72cfb96b9a51f6945a8126c9b173ff0b0a"
+				}
+			]
+	}],
+	"finish-args": [
+		"--env=PATH=/app/jre/bin:/usr/bin",
+    	"--share=ipc",
```

Comment:
nitpick: indenting, also nitpick, we generally have finish args at the top of the file as it makes it a bit more consistent to audit permissions.